src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -998,6 +998,12 @@ def set_tag(self, key, value):
         )
         self.tags[key] = value
 
+    def deactivate(self):
+        """
+        Deactivate the LV by calling lvchange -an
+        """
+        process.call(['lvchange', '-an', self.lv_path])
```

Comment:
I haven't tried de-activating the LV only, unsure if it's sufficient.

Reply:
well at the very least it's needed as a first step before deactivating a VG.